extensions/amp-subscriptions-google/amp-subscriptions-google.md

```diff
@@ -115,7 +115,7 @@ Where `data` matches the [entitlements response](https://github.com/subscription
           }
         },
         {
-          "serviceId": "subscribe.google.com"
+          "serviceId": "subscribe.google.com",
```

Comment:
Please remove trailing comma. The JSON parsers are very strict in this regard. So we have to be very careful with our examples.

Reply:
I hate that of JSON parsers :( always comes to bit me at some point :(

Done!